actions/websocket_actions.jsx

```diff
@@ -188,7 +188,7 @@ function handleFirstConnect() {
 
 function handleClose(failCount) {
     if (failCount > MAX_WEBSOCKET_FAILS) {
-        ErrorStore.storeLastError({message: ErrorBarTypes.WEBSOCKET_PORT_ERROR});
+        ErrorStore.storeLastError({message: AnnouncementBarMessages.WEBSOCKET_PORT_ERROR});
```

Comment:
Should we backport this into 5.4, since I'm guessing this was a regression?

Reply:
Yes, you are right. It is recent. Usually when I find these things they have been there forever. When this get's merged I will do a cherry pick of these fixes to 5.4.